.travis.yml

```diff
@@ -13,6 +13,7 @@ before_cache:
 env:
   global:
     - MAKEFLAGS="-j 2"
+    - CMAKE_GENERATOR="Unix Makefiles"
```

Comment:
Is this just adding a default generator?

Reply:
Yea.